app/lib/widgets/location_overview.dart

```diff
@@ -1,8 +1,8 @@
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
+import 'package:manvsim/constants/manv_icons.dart';
 import 'package:manvsim/models/location.dart';
 
-import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import 'package:manvsim/widgets/media_overview_expansion.dart';
 
 class LocationOverview extends StatefulWidget {
```

Comment:
Seems like we don't need State here either

Reply:
Removed